opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -672,16 +672,64 @@ bool FieldProps::supported<int>(const std::string& keyword) {
     return Fieldprops::keywords::isFipxxx(keyword);
 }
 
+void FieldProps::apply_multipliers()
+{
+    static const auto prefix = FieldPropsManager::getMultiplierPrefix();
+
+    for(auto pos =multiplier_kw_infos_.begin(); pos != multiplier_kw_infos_.end(); ++pos)
+    {
+        const auto& [mult_keyword, kw_info] = *pos;
+        const std::string keyword = mult_keyword.substr(prefix.size());
+        auto mult_iter = this->double_data.find(mult_keyword);
+        assert(mult_iter != this->double_data.end());
+        auto iter = this->double_data.find(keyword);
+        if (iter == this->double_data.end()) {
+            iter = this->double_data
+                .emplace(std::piecewise_construct,
+                         std::forward_as_tuple(keyword),
+                         std::forward_as_tuple(kw_info, this->active_size, kw_info.global ? this->global_size : 0))
+                .first;
+        }
+        auto multiplier = mult_iter->second.data.begin();
+        for(auto data = iter->second.data.begin(); data != iter->second.data.end(); ++data, ++multiplier)
+        {
+            *data *= *multiplier;
+        }
```

Comment:
I think I'd probably use `std::transform()` for this.  Something along the lines of
```C++
std::transform(iter->second.data.begin(), iter->second.data.end(),
               mult_iter->second.data.begin(),
               iter->second.data.begin(), std::multiplies<>{});
```